docs/reference/hooks.rst

```diff
@@ -614,6 +614,31 @@ Hooks for customising the way users are directed through the process of creating
 
   Uses the same behaviour as ``before_create_page``.
 
+
+.. _before_convert_alias_page:
+
+``before_convert_alias_page``
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Just noticed from CI that these underlines need extending.

Reply:
Oops. fixed!